spec/lib/requirements/topic_checker_spec.rb

```diff
@@ -25,26 +27,28 @@
       end
 
       it "returns an issue if there are no topics" do
-        issues = Requirements::TopicChecker.new(document).pre_publish_issues
+        issues = Requirements::TopicChecker.new(edition.document).pre_publish_issues
         expect(issues).to have_issue(:topics, :none, styles: %i[form summary])
       end
     end
 
     context "when the Publishing API is down" do
-      let(:document_type) { build :document_type, topics: true }
-      let(:document) { build :document }
+      let(:document_type) { create :document_type, topics: true }
+      let(:metadata_revision) { create :metadata_revision, document_type_id: document_type.id }
+      let(:revision) { create :revision, metadata_revision: metadata_revision }
+      let(:edition) { create :edition, revision: revision }
```

Comment:
Could these `let`s be moved to just after the `describe` block on line 6, and reused instead of duplicated by lines 14-17? The first test should continue to pass since you're defining an issueless `edition` 'inline'.